test/tools/libtesteth/TestSuite.cpp

```diff
@@ -31,17 +31,28 @@ namespace dev
 namespace test
 {
 
-void TestSute::runAllTestsInFolder(const std::string& _folder) const
+// Structure  <suiteFolder>/<testFolder>/<test>.json
+// Return full path to folder for tests from _testFolder
+fs::path getFullPathFiller(string const& _suiteFolder, string const& _testFolder)
```

Comment:
It's better to define these inside anonymous namespace (`namespace {   } `) instead of `test`, because they're internal for this file.

If they're in `test` it can lead to name conflicts during link-time.